src/modules/Builder/features/PerformanceTasks/Unity/UnityFilePreview/UnityFilePreview.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+interface UnityFilePreviewProps {
+  fileContent: string;
+}
+
+export const UnityFilePreview: React.FC<UnityFilePreviewProps> = ({ fileContent }) => (
```

Comment:
You mentioned creating a new ticket to complete the UI of `UnityFilePreview`, please add this as a TODO in this file